package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "blob-to-buffer",
   "description": "Convert a Blob to a Buffer",
-  "version": "1.2.8",
+  "version": "2.0.0",
```

Comment:
Please don't bump the version in a PR. This should happen when the new version is actually published.

Reply:
sure